net-libs/shairplay/shairplay-0_pre20160304.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+
+inherit autotools eutils
+
+if [[ ${PV} == "9999" ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="git://github.com/juhovh/shairplay.git"
+else
+	EGIT_COMMIT="498bc5bcdd305e04721f94a04b9f26a7da72673f"
+	SRC_URI="https://github.com/juhovh/shairplay/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/${PN}-${EGIT_COMMIT}"
+fi
+
+DESCRIPTION="Apple airplay and raop protocol server"
+HOMEPAGE="https://github.com/juhovh/shairplay"
+
+LICENSE="BSD LGPL-2.1 MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="static-libs"
+
+RDEPEND="
+	>=dev-libs/openssl-1.0.0
+	media-libs/libao
+"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		$(use_enable static-libs static)
+}
+
+src_install() {
+	default
+	prune_libtool_files --all
```

Comment:
Replace this with:
```
find "${D}" -name '*.la' -delete || die
```

For rationale, see: https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Handling_Libtool_Archives